tests/test_tracing.py

```diff
@@ -1841,8 +1895,18 @@ def test_extract_dd_trace_context_for_eventbridge(self):
             event = json.load(event)
         ctx = get_mock_context()
         context, source, event_type = extract_dd_trace_context(event, ctx)
-        self.assertEqual(context["trace-id"], "12345")
-        self.assertEqual(context["parent-id"], "67890")
+        self.assertEqual(context.trace_id, 12345)
+        self.assertEqual(context.span_id, 67890)
+
+    def test_extract_dd_trace_context_for_w3c(self):
+        event_sample_source = "lambda-url-w3c"
+        test_file = event_samples + event_sample_source + ".json"
+        with open(test_file, "r") as event:
+            event = json.load(event)
+        ctx = get_mock_context()
+        context, source, event_type = extract_dd_trace_context(event, ctx)
+        self.assertEqual(context._traceparent, "00-00000000000000005f486460f346b80c-dc070c2bb8714351-01")
+        self.assertEqual(context._tracestate, "dd=s:1;t.dm:-1")
```

Comment:
Let's actually test the trace id, span id, and sampling priority on the context directly. Attributes starting with an `_` in Python are implied to be private and could change at anytime.